Common/VsDevHelper.cs

```diff
@@ -85,11 +85,12 @@ private static string FindVsDevCmd()
             if (programFiles == null)
                 return null;
 
-            foreach (var version in Helper.VisualStudioVersions())
+            foreach (var version in Helper.VisualStudioVersions)
+            foreach (var edition in Helper.VisualStudioEditions)
             {
                 paths.Add(new KeyValuePair<string, string>(
                     "VS150COMNTOOLS",

```

Comment:
Does vs2019 have such variable with greater number, like `VSxxxCOMNTOOLS`?

Reply:
```
D:\Develop\cement
λ set | grep -i comntools
VS110COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio 11.0\Common7\Tools\
VS120COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio 12.0\Common7\Tools\
VS140COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\
VS160COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\Common7\Tools\
```
but only when you run a Developer Command Prompt